src/pages/WorkspaceDetails/OverviewTab/StorageType/index.tsx

```diff
@@ -93,9 +92,9 @@ export class StorageTypeFormGroup extends React.PureComponent<Props, State> {
   }
 
   private getExistingTypes(): { hasAsync: boolean, hasPersistent: boolean, hasEphemeral: boolean } {
-    const hasAsync = this.storageTypes.some(type => StorageType[type] === StorageType.async);
-    const hasPersistent = this.storageTypes.some(type => StorageType[type] === StorageType.persistent);
-    const hasEphemeral = this.storageTypes.some(type => StorageType[type] === StorageType.ephemeral);
+    const hasAsync = this.storageTypes.some(type => type === 'async');
```

Comment:
Why do you remove the common ('/services/helpers/types' ) enum?

Reply:
If I got the question correctly then the answer is [here](https://github.com/che-incubator/che-dashboard-next/pull/83#discussion_r530160111).
If not - please elaborate.

Reply:
ok

Reply:
I explain my approach [here](https://github.com/che-incubator/che-dashboard-next/pull/83#discussion_r530206506).